jscomp/gentype_tests/typescript-react-example/src/ExportWithRename.gen.tsx

```diff
@@ -5,9 +5,8 @@
 
 import * as React from 'react';
 
-import * as ExportWithRenameBS__Es6Import from './ExportWithRename.bs';
-const ExportWithRenameBS: any = ExportWithRenameBS__Es6Import;
+import * as ExportWithRenameBS from './ExportWithRename.bs';
 
 export type Props = { readonly s: string };
 
-export const make: React.ComponentType<{ readonly s: string }> = ExportWithRenameBS.ExportWithRename;
```

Comment:
TypeScript report type error on this

```
Property 'ExportWithRename' does not exist on type 'typeof import(".../ExportWithRename.bs")'.
```

Which is accurate.

That may be the intended behavior of `@gentType.as`, but it doesn't actually work. If it's not a bug, I can't even fix it.

Reply:
Fixed in https://github.com/rescript-lang/rescript-compiler/pull/6509